src/core/Akka/Dispatch/Dispatchers.cs

```diff
@@ -255,34 +318,42 @@ private Config IdConfig(string id)
             return ConfigurationFactory.ParseString(string.Format("id: {0}", id));
         }
 
+
+        private static readonly Config ForkJoinExecutorConfig = ConfigurationFactory.ParseString("executor=fork-join-executor");
```

Comment:
Configuration stubs used for backwards compatibility with previous dispatcher declarations. Going to integration test this backwards-compat by installing a nightly into Akka.NET bootcamp and making sure all of the synchronized dispatchers there still run.